python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -39,6 +39,14 @@ def test_licence(self):
         errors = tidy.collect_errors_for_files(iterFile('incorrect_license.rs'), [], [tidy.check_license])
         self.assertEqual('incorrect license', errors.next()[2])
 
+    #def test_modeline(self):
```

Comment:
yes, this is the other half of https://github.com/edunham/servo/blob/tidy-licensecheck/python/tidy/servo_tidy/tidy.py#L126 which I figured I should leave to make a quicker start for whoever works on https://github.com/servo/servo/issues/10719.  The lint can't be enabled yet because we actually want to keep modelines in `/tests/wpt` as discussed in https://github.com/servo/servo/pull/10723